src/chrome/locale/ar/https-everywhere.dtd

```diff
@@ -8,12 +8,14 @@
 <!ENTITY https-everywhere.about.contribute  "إن أعجبتك إضافة HTTPS Everywhere، فكر بـ">
 <!ENTITY https-everywhere.about.donate_tor "التبرع لتور">
 <!ENTITY https-everywhere.about.tor_lang_code "en">
+<!ENTITY https-everywhere.about.or "or">
```

Comment:
Yeah, I actually tried that since that's what I gathered from the Transifex manual, but makexpi gave a warning without them.